docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,195 @@
+
+test('MyTest', async t => {
+    await t
+        .click('.populate-form')
+        .handleConfirm('OK')
+        .click('#submit-button');
+
+    expect(await t.eval( () => document.querySelector('#article-header').text )).to.equal('Thank You, Peter Parker!');
+};
+```
+
+Besides, TestCafe API provides special methods to handle other native dialogs such as alerts and prompts. For more information, see [Handling Native Dialog](../test-api/handling-native-dialogs.md).
+
+### Authentication
+
+At this step, you will see how TestCafe handles webpage authentication.
+
+The subsequent test clicks the Authentication button that performs Http Basic authentication, opens a Personal Cabinet and checks whether the user name is correct. To provide the user credentials, the `httpAuth` function is used.
+
+```js
+fixture `Getting Started - Test 4`
+    .page('http://testcafe.devexpress.com/Example')
+    .httpAuth('test', '1234');
+
+test('MyTest', async t => {
+    await t
+        .click('.populate-form')
+        .handleConfirm('OK')
+        .click('.personal-cabinet-button');
+
+    expect(await t.eval( () => document.querySelector('.user-name').text )).to.equal('Peter Parker');
+};
+```
+
+See [Http Authentication](../test-api/http-authentication.md) to learn more about authentication.
+
+## Running Test
+
+You can simply run tests from a command shell, by calling a single command where you specify target browsers and tests.
+
+```bash
+testcafe ie,chrome my-project/tests/test1.js
+```
+
+TestCafe will automatically open the chosen browsers and start test execution within them simultaneously.
+
+You can run tests in [local browsers](../using-testcafe/command-line-interface.html#local-browsers) as well as on [remote devices](../using-testcafe/command-line-interface.html#remote-browsers) (including mobile phones and tablets).
+The only requirement for a device to run TestCafe tests is that it should have network access to the TestCafe application.
+
+TestCafe provides a number of additional options for the `testcafe` command to configure test run.
+Among them, there are options that allow you to enable screenshot capturing, specify a reporter to generate reports, enable the quarantine mode, etc.
+For more information, see [Options](../using-testcafe/command-line-interface.md#options).
+
+## Viewing Test Results
+
+While a test is running, TestCafe is gathering information about the test run and outputing it right into a command shell. After the test run has finished, you can view a complete report there.
+
+IMAGE
+
+If errors occurred during test run, a report includes detailed information about them. Fancy call sites, clean stacks and screenshots will help you easily detect the cause of an error.
+
+Test reports are generated by TestCafe [reporters](../using-testcafe/common-concepts/reporters.md). By default, the spec reporter is used, but you can select any other built-in reporter or create and use your own custom reporter.
```

Comment:
`Test reports are generated by TestCafe [reporters](../using-testcafe/common-concepts/reporters.md).` you already said that in different words above.